### PR TITLE
Update content across different pages

### LINKED ISSUE #9: Update website's content with actual content
The website design currently contains dummy text/data, this is to update the website pages with actual content (Text and Images).

- [x] Get in touch with the content team to inform them of the required content needed on the website
- [x] Get in touch with the creative team to inform them of the required assets still needed for the website
- [x] Update dummy text with actual data and replace resources/assets where necessary

### PR DESCRIPTION
**Pull request by yemisi-o**
Updated dummy text and some previous content with updated text

Closes #9

### CODE REVIEW

File: templates/views/partners.html [hunk 1]
```diff
@@ -13,8 +13,10 @@
     <div class="container">
         <br><br>
         <p class="__shecodeval">Why <span class="__shecodeval1">Partner</span> With Us</p>
-        <div class="text-center">
-            Lorem ipsum dolor sit amet consectetur adipisicing elit. Sint laborum, aliquid molestiae ut error amet itaque fugiat deserunt aliquam reprehenderit cumque exercitationem eveniet ex debitis veritatis iste, quasi magni nesciunt.
+        <div class="community_text">
+            Partner with us and help us make an impact!  Your donation would go a long way in securing and 
+            covering costs for our numerous event and offline initiatives, providing learning resources and
```
Comment (mazma1) [inline templates/views/partners.html]: ...our numerous event**s**

File: templates/views/index.html [hunk 2]
```diff
@@ -46,37 +46,41 @@
         </div>
 
         <div class="row __shecodebestie">
-            <h2 class="text-center">Become our bestie...</h2>
+            <h2 class="text-center">Several ways to Identify with us</h2>
```
Comment (mazma1) [inline templates/views/index.html]: ...Identify with **U**s

File: templates/views/partners.html (hunk 1 above)
Comment (yemisi-o) [reply templates/views/partners.html]: Thanks, will update

File: templates/views/index.html (hunk 2 above)
Comment (yemisi-o) [reply templates/views/index.html]: Thanks, will update

Comment (yemisi-o) [reply templates/views/index.html]: Updated

File: templates/views/partners.html (hunk 1 above)
Comment (yemisi-o) [reply templates/views/partners.html]: Updated

**Pull request merged**